core/templates/pages/topic-editor-page/editor-tab/topic-editor-tab.directive.ts

```diff
@@ -334,16 +334,31 @@ angular.module('oppia').directive('topicEditorTab', [
             }
           };
 
+          // Only update the topic if 1) the creator is turning off the
+          // practice tab or 2) the creator is turning on the practice tab
+          // and it has enough practice questions.
           $scope.updatePracticeTabIsDisplayed = function(
               newPracticeTabIsDisplayed) {
-            if (
-              newPracticeTabIsDisplayed !==
-              $scope.topic.getPracticeTabIsDisplayed()) {
+            if (!newPracticeTabIsDisplayed ||
+              $scope.topicHasMinimumQuestionsToPractice()
+            ) {
               TopicUpdateService.setPracticeTabIsDisplayed(
                 $scope.topic, newPracticeTabIsDisplayed);
+              $scope.editablePracticeIsDisplayed = newPracticeTabIsDisplayed;
             }
           };
 
+          $scope.topicHasMinimumQuestionsToPractice = function() {
```

Comment:
Maybe rename to `doesTheTopicHaveMinimumPracticeQuestions` or something like that which starts with a verb? Since, currently it sounds like a variable and not a function.

Reply:
done